src/renderer/components/node/Node.tsx

```diff
@@ -297,7 +297,9 @@ const NodeInner = memo(({ data, selected }: NodeProps) => {
             targetRef={targetRef}
             toggleCollapse={toggleCollapse}
             validity={validity}
-            onContextMenu={menu.onContextMenu}
+            onContextMenu={(e) => {
+                menu.onContextMenu(e);
+            }}
```

Comment:
I think we can revert this.

```suggestion
            onContextMenu={menu.onContextMenu}
```